### ISSUE TITLE
Execution of aspire module as a script

### ISSUE THREAD

**Issue by vineetbansal** [ComputationalCryoEM/ASPIRE-Python#19]
Support execution of the entire module as a script, something like:

```
python -m aspire pick --starfile mystarfile.star
python -m aspire classify --stackfile mymrcs.mrcs
```
etc.

The top-level catch-all script should most likely support the `click` library, and delegate the work to `Commands` in a separate `commands` submodule. Might need to see how other libraries do this.

**Comment by vineetbansal** [comment]
Likely complete, but should not be closed till #21 is fixed.

**Comment by vineetbansal** [comment]
fixed in b219c5684fec58dad66f823538942536b4f49b12
